### ISSUE TITLE
where do i put the factory presets so that they show up in the vst?

### ISSUE THREAD

**Issue by billyblackburn** [mtytel/helm#123]
my issue was resolved and it was my mistake to begin with.  please disregard this "issue"

**Comment by MrDorianJames** [comment]
I'm actually having problems with this. 
I switched over to Ubuntu recently and love your plugin and used it with my mac.
I first installed it using the .deb file on your site but quickly realized that the .deb only has the standalone in it. I'd like to use it in Tracktion or Renoise.

 I finally saw that KXstudio has helm in their repo.  However, when I installed that, there was no presets  and i can't save any presets either. I've tried the import bank on the browse page but it's looking for .helmbanks and not patches. I've tried loading some third party _.helmbank_  files and they don't work either. I can't seem to import individuals presets anywhere on the software either. 
i've moved some files around to the different locations to see if my i had things in wrong file locations but nothing

> **GNU/Linux:**
> Standalone - /usr/bin/helm.
> LV2 - /usr/lib/lv2/helm.lv2
> VST - /usr/lib/lxvst/helm.so (if installed).
> Local Presets - ~/.helm/patches
> Factory Files - /usr/share/helm/
> Config File - ~/.helm/Helm.config

so then I thought maybe I was being lazy and I should build it from source. I downloaded the zip from your github page and tried compiling it but i ran into this error
```
In file included from ../../JuceLibraryCode/JuceHeader.h:29,
                 from ../../../src/common/border_bounds_constrainer.h:21,
                 from ../../../src/common/border_bounds_constrainer.cpp:17:
../../../JUCE/modules/juce_opengl/juce_opengl.h:92:11: fatal error: GL/gl.h: No such file or directory
   92 |  #include <GL/gl.h>
      |           ^~~~~~~~~
compilation terminated.
make[1]: *** [Makefile:399: build/intermediate/Release/border_bounds_constrainer_b5a34af8.o] Error 1
make[1]: Leaving directory '/home/dorianjames/Downloads/helm-0.9.0/standalone/builds/linux'
make: *** [Makefile:80: standalone] Error 2
```

This was a resolved from [issue #60](https://github.com/mtytel/helm/issues/60) and from what i could understand, it was fixed in the development branch so I've tried to build from the dev source with the same problem. I'm kind of out of ideas at this point. 

Do you have any suggestions?

**Comment by andrewsaraceno** [comment]
Looks like you never got an answer. Im having the exact same problem and I contacted the developer Matt Tytel (i blieve is his name) but he never replied. Glad i didn't donate any money to the Helm synh only to get a completely empty synth after donating.
